src/groundStation/ftp.py

```diff
@@ -112,6 +118,7 @@ def receive_burst(self, req_id, skip, count):
             # I know it's not good to hardcode the byte I want like this
             # but there's too much legacy so it won't change
             if data[0] == self.vals.SERVICES.get("FTP_COMMAND").get('subservice').get('FTP_DATA_PACKET').get('subPort'):
+                print("Expecting blocK: {}, Got block: {}".format(blockcount, rxDataList[0]['blocknum']))
```

Comment:
Did you mean to leave this in? I notice the check that follows prints a block number if it was missed.

Reply:
Oh yeah I didn't mean to leave that in